src/pages/404.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Heading, Stack, VStack, Text } from '@chakra-ui/react'
+import { NextSeo } from 'next-seo'
+import Image from 'next/image'
+import NextLink from 'next/link'
+
+import mascotLogo from '@app-public/mascot-logo.png'
+import { CTAButton } from '~/components/CTAButton'
+import FluidContainer from '~/components/FluidContainer'
+import Layout from '~/components/Layout'
+import Link from '~/components/Link'
+
+const Custom404 = () => (
+  <Layout>
+    <NextSeo title="404 - Not Found" />
+    <Box pb={{ base: 8, lg: 16 }} align="center">
+      <FluidContainer>
+        <VStack px="10">
+          <Image
+            src={mascotLogo}
+            alt="Octoclairvoyant reading a crystal ball"
+            placeholder="blur"
+            priority
+            width={250}
+            height={250}
+          />
+          <Stack
+            shouldWrapChildren
+            alignItems="center"
+            spacing="2"
+            flexGrow={1}
+            flexBasis={0}
+          >
+            <Heading
+              as="h2"
+              color="gray.900"
+              fontSize="4xl"
+              fontWeight="black"
+              letterSpacing="tight"
+            >
+              This page could not be found.
+            </Heading>
+            <Text as="p" color="coolGray.700" fontSize="2xl" fontWeight="black">
+              Octoclairvoyant cannot divine the page you wanted.
+            </Text>
+          </Stack>
+        </VStack>
+        <VStack mt={12} spacing={5}>
+          <NextLink href="/comparator">
+            <CTAButton>Go to comparator</CTAButton>
+          </NextLink>
+
+          <NextLink href="/">
+            <Link fontWeight="black">Or go to homepage</Link>
```

Comment:
For some reason I cannot use the keyboard with <kbd>tab</kbd> to focus on the `Or go to homepage` text.

Tabbing sequence:

1. `h1` in navbar
1. Developer `Toggle Dark` button
1. Big fat button `Go to comparator`
1. Skips over `Or go to homepage` text completely
1. Footer `Mario`
1. Footer `GitHub`
1. Vercel badge

Reply:
I don't know why this happens from time to time. It's the same issue you mentioned a few days ago about not being able to focus some links in the feature section of the homepage. I was able to do so on my other laptop, so there must be something preventing us to do so. I'll investigate it.

Reply:
Oh wow, the E2E test caught this too! I wasn't passing the href from the Next link to the Chakra link, ending in a wrong link from an a11y point of view. It should be fixed now!